src/test/java/com/google/sps/ListOrgServletTest.java

```diff
@@ -0,0 +1,197 @@
+    //                  Existing entity table
+    // +-------+-------------------------+------------+---------+
+    // | Index | creationTimeStampMillis | isApproved | zipCode |
+    // +-------+-------------------------+------------+---------+
+    // |     0 |                       0 | false      |   12345 |
+    // |     1 |                       1 | true       |   02763 |
+    // |     2 |                       2 | true       |   47906 |
+    // |     3 |                       3 | false      |   47906 |
+    // |     4 |                       4 | true       |   02763 |
+    // |     5 |                       5 | false      |   02763 |
+    // |     6 |                       6 | false      |   94566 |
+    // +-------+-------------------------+------------+---------+
+
+    Entity entity0 = new Entity("Distributor");
+    entity0.setProperty("creationTimeStampMillis", 0);
+    entity0.setProperty("isApproved", false);
+    entity0.setProperty("orgZipCode", "12345");
+    masterEntityList.add(entity0);
+    datastore.put(entity0);
+ 
+    Entity entity1 = new Entity("Distributor");
+    entity1.setProperty("creationTimeStampMillis", 1);
+    entity1.setProperty("isApproved", true);
+    entity1.setProperty("orgZipCode", "02763");
+    masterEntityList.add(entity1);
+    datastore.put(entity1);
+ 
+    Entity entity2 = new Entity("Distributor");
+    entity2.setProperty("creationTimeStampMillis", 2);
+    entity2.setProperty("isApproved", true);
+    entity2.setProperty("orgZipCode", "47906");
+    masterEntityList.add(entity2);
+    datastore.put(entity2);
+ 
+    Entity entity3 = new Entity("Distributor");
+    entity3.setProperty("creationTimeStampMillis", 3);
+    entity3.setProperty("isApproved", false);
+    entity3.setProperty("orgZipCode", "47906");
+    masterEntityList.add(entity3);
+    datastore.put(entity3);
+ 
+    Entity entity4 = new Entity("Distributor");
+    entity4.setProperty("creationTimeStampMillis", 4);
+    entity4.setProperty("isApproved", true);
+    entity4.setProperty("orgZipCode", "02763");
+    masterEntityList.add(entity4);
+    datastore.put(entity4);
+ 
+    Entity entity5 = new Entity("Distributor");
+    entity5.setProperty("creationTimeStampMillis", 5);
+    entity5.setProperty("isApproved", false);
+    entity5.setProperty("orgZipCode", "02763");
+    masterEntityList.add(entity5);
+    datastore.put(entity5);
+ 
+    Entity entity6 = new Entity("Distributor");
+    entity6.setProperty("creationTimeStampMillis", 6);
+    entity6.setProperty("isApproved", false);
+    entity6.setProperty("orgZipCode", "94566");
+    masterEntityList.add(entity6);
+    datastore.put(entity6);
+  }
+ 
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+ 
+  @Test
+  public void testQueryWithNoFilter() {
+    /* For this test, all of the entities need to be returned because there are no filter limits,
+     * but the master list must be reversed since the query returns them in descending order*/
```

Comment:
I think you can simplify this comment to:

Because there are no filters, all entities are returned in reverse time order.